sim/deathknight/talents_blood.go

```diff
@@ -295,12 +295,6 @@ func (dk *Deathknight) applySuddenDoom() {
 
 	procChance := 0.05 * float64(dk.Talents.SuddenDoom)
 
-	dk.SuddenDoomAura = dk.RegisterAura(core.Aura{
-		Label:    "Sudden Doom Proc",
-		ActionID: core.ActionID{SpellID: 49530},
-		Duration: core.NeverExpires,
-	})
```

Comment:
This is used in the result metrics to know how many times you've procced the aura so it should be reverted even if not used by the GCD/Cost skip mechanics anymore

Reply:
Put this back too